textpipe/pipeline.py

```diff
@@ -37,3 +38,37 @@ def __call__(self, raw):
         doc = Doc(raw, language=self.language, hint_language=self.hint_language)
         result_dict = dict(zip(self.pipeline, [doc.__getattribute__(x) for x in self.pipeline]))
         return result_dict
+
+    def save(self, filename):
+        """
+        Serializes the pipeline to file (using json)
+
+        Args:
+        filename: location of where to serialize pipeline
+
+        >>> filename = "test.json"
+        >>> Pipeline(['n_sents', 'clean_text']).save(filename)
+        >>> open(filename).read()
+        '{"pipeline": ["n_sents", "clean_text"], "language": null, "hint_language": null}'
```

Comment:
This might fail, the `self.__dict__` doesn't have a defined order.

Reply:
Solved that here like this: https://github.com/textpipe/textpipe/blob/master/textpipe/pipeline.py#L13.